src/pages/index.js

```diff
@@ -38,39 +89,83 @@ const profileEditModal = new PopupWithForm({
   },
 });
 
+profileEditModal.setEventListeners();
+
 const addCardModal = new PopupWithForm({
   popupSelector: "#card-edit-modal",
   handleFormSubmit: (data) => {
     console.trace(data);
+    addCardModal.setLoading(true);
+    api
+      .createCards(data)
+      .then((res) => {
+        cardSection.addItem(createCard(res));
+        Constants.cardEditForm.reset();
+        addFormValidator.resetValidation();
+      })
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**